package.json

```diff
@@ -53,7 +53,7 @@
     "cy": "CYPRESS_dexUrl=https://$INGRESS_HOST:$PORT_HTTPS CYPRESS_baseUrl=http://localhost:9999 cypress open",
     "cy:dev": "source ../monitor-ci/.env && CYPRESS_dexUrl=CLOUD CYPRESS_baseUrl=https://$INGRESS_HOST:$PORT_HTTPS cypress open --config testFiles='{cloud,shared}/**/*.*'",
     "cy:dev-oss": "source ../monitor-ci/.env && CYPRESS_dexUrl=OSS CYPRESS_baseUrl=https://$INGRESS_HOST:$PORT_HTTPS cypress open --config testFiles='{oss,shared}/**/*.*'",
-    "generate": "export SHA=689356a1c8e3dbd4e6ff6783cf81cbb720f8e64a && export REMOTE=https://raw.githubusercontent.com/influxdata/openapi/${SHA}/ && yarn generate-meta",
+    "generate": "export SHA=db14de0d91be00e75eb946d768e88d771b20a2a9 && export REMOTE=https://raw.githubusercontent.com/influxdata/openapi/${SHA}/ && yarn generate-meta",
```

Comment:
One thing I noticed in testing this locally: each of the three options for the `query` limits is a specific number, `0`, -`-1`, or anything else. But in the openapi spec that's generated, they're just 'any number'. So we should fix that so that it's `0` or `-1` or `number`.

Reply:
<img width="747" alt="Screen Shot 2023-02-15 at 9 00 01 AM" src="https://user-images.githubusercontent.com/66275100/219064430-1e2c87b2-409f-43dc-8487-ec621b7cbd49.png">
For timeout limits, the subfields should be of types `RestrictedLimit` `Unlimited` and `Limit`, which map to the numbers you mentioned. Check out the screenshot.

Reply:
Correct, but check what those map to - it's just a `number`.